### PR TITLE
Pb bug bash 28 03 2024

### PR DESCRIPTION
**Pull request by Pbarbosa92**
Some minor fixes detected during a bug bash.

### CODE REVIEW

File: docs/explore_api/templating.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ machine_group = inductiva.resources.MachineGroup(machine_type="c2-standard-30")
 machine_group.start()
 
 # Download the input files mentioned above
-input_dir = inductiva.utils.download_from_url(
+template_dir = inductiva.utils.download_from_url(
```
[the remark this answers is not in the document]
Comment (Pbarbosa92) [reply docs/explore_api/templating.md:190]: You are correct. I will Revert my changes on this file.

**Review by AugustoPeres — APPROVED** [review]
LGTM

File: docs/Home.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ computational resources and checking the status of tasks;
 - A [User Reference](/api_reference/computational_resources/index.md) section 
 that covers a wide variety of topics of interest, including information about
 some key classes available in the API Client, a troubleshooting guide, information
-about quotas and an FAQ.
+about quotas and an [FAQ](api_reference/faq.md).
```
Comment (AugustoPeres) [inline docs/Home.md:30]: "a [FAQ]"

Comment (Pbarbosa92) [reply docs/Home.md:30]: "An FAQ" is correct.

Comment (AugustoPeres) [reply docs/Home.md:30]: You are correct

File: docs/how_to/run-parallel_simulations.md [hunk 3]
```diff
@@ -47,8 +47,8 @@ water_levels_list = [3.5, 3.75, 4.0, 4.5, 5.0]
 # Launch multiple simulations for each water level
 for water_level in water_levels_list:
     # Set the root directory and render the template files into it.
-    template_engine = inductiva.TemplateEngine("swash-input-example")
-    template_engine.add_dir(input_dir, water_level=water_level)
+    template_engine = inductiva.TemplateManager(template_dir="swash-input-example")
```
Comment (IvanPombo) [inline docs/how_to/run-parallel_simulations.md:50]:
```suggestion
    template_manager = inductiva.TemplateManager(template_dir=template_dir, root_dir="swash-example")
```

File: docs/how_to/run-parallel_simulations.md [hunk 4]
```diff
@@ -47,8 +47,8 @@ water_levels_list = [3.5, 3.75, 4.0, 4.5, 5.0]
...
+    template_engine.render_dir(water_level=water_level)
```
Comment (IvanPombo) [inline docs/how_to/run-parallel_simulations.md:51]:
```suggestion
    template_manager.render_dir(water_level=water_level)
```

File: docs/how_to/run-parallel_simulations.md [hunk 5]
```diff
@@ -47,8 +47,8 @@ water_levels_list = [3.5, 3.75, 4.0, 4.5, 5.0]
...
     
     # Run simulation with the configured input directory on the dedicated MachineGroup
     task = swash.run(input_dir=template_engine.get_root_dir(),
```
Comment (IvanPombo) [inline docs/how_to/run-parallel_simulations.md:54]:
```suggestion
    task = swash.run(input_dir=template_manager.get_root_dir(),
```

**Review by IvanPombo — CHANGES_REQUESTED** [review]
This is a first round...

Same here: https://github.com/inductiva/inductiva/blob/pb-bug-bash-28-03-2024/docs/how_to/set-up-elastic-machine-group.md
and check over the rest.

**Review by IvanPombo — CHANGES_REQUESTED** [review]
Indeed you have reacted to my comments, but it is still missing tackling the code in the link I have also sent :D

**Comment by Pbarbosa92** [comment]
> Indeed you have reacted to my comments, but it is still missing tackling the code in the link I have also sent :D

I think this time it's all. There is the Templating.md file missing, but that was addressed by Sergio on https://github.com/inductiva/inductiva/pull/1249#event-12283748016

**Review by IvanPombo — APPROVED** [review]
LGTM!

**Pull request merged**